### ISSUE TITLE
CNI: macvlan plugin does not work because of missing module

### ISSUE THREAD

**Issue by fabiand** [kubernetes/minikube#1564]
**BUG REPORT**:

**Minikube version** (use `minikube version`):
$ minikube version
minikube version: v0.19.0

**Environment**:
- **OS** (e.g. from /etc/os-release): VERSION="25 (Workstation Edition)"
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): kvm
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): v0.18.0
- **Install tools**:
- **Others**:

**What happened**:
Tried to setup CNI with macvtap

**What you expected to happen**:
That it worked

**How to reproduce it** (as minimally and precisely as possible):
```bash
    ip link add name peth0 link eth0 type macvlan
    RTNETLINK answers: Operation not supported
```

**Anything else do we need to know**:
Can we add the macvlan kmod to the iso?
I suppose that's all that is missing

**Comment by r2d4** [comment]
Yeah, that sounds fine. 

We've had some trouble in the past with CNI implementations in the past since a lot assume different modules are installed.  Would be great to have a more definitive list so that we can enable all the ones that we need.

**Comment by fabiand** [comment]
Right - maybe that list is actually something which should be maintained on the CNI side - I'll see if I can come up with it.

**Comment by dlorenc** [comment]
Fixed with #1600
